src/main/java/fr/insalyon/citi/golo/compiler/ParseTreeToGoloIrVisitor.java

```diff
@@ -35,6 +36,10 @@
 
   private GoloCompilationException.Builder exceptionBuilder;
 
+  private static String syntheticName(String base) {
+    return "__$$_" + base + "_" + System.currentTimeMillis();
```

Comment:
*"However, since I don't know when it will be ready, I can already make change (simply using a counter) in this PR."*
:+1: